spec/cc/engine/gem_detector_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "spec_helper"
+require "cc/engine/gem_detector"
+
+module CC::Engine
+  describe GemDetector do
+    describe "#present?" do
+      before do
+        @detector = GemDetector.new("./spec/fixtures/Gemfile.lock")
+      end
+
+      it "returns true when spec is present in Gemfile.lock" do
+        @detector.present?("ansi").must_equal(true)
+        @detector.present?("i18n").must_equal(true)
+        @detector.present?("astrolabe").must_equal(true)
```

Comment:
I consider it a bit of a smell when you have a nice predicate method but then resort to checking equality with `true`/`false` in your specs. I think things would read better like this:

```rb
specs = LockfileSpecs.new("./spec/fixures/Gemfile.lock")

expect(specs).to include("ansi")
expect(specs).not_to include("pants")
```

Renaming something for the specs alone isn't always the best thing to do, but in this case I think it'd be more readable in the implementation code as well.

What do you think about renaming `GemDetector` => `LockfileSpecs` and `present?` => `include?`. I think that's all it'd take.